.github/actions/xahau-ga-build/action.yml

```diff
@@ -76,6 +88,22 @@ runs:
           export CXX="${{ inputs.cxx }}"
         fi
         
+        # Set ccache configuration
+        mkdir -p ~/.ccache-main ~/.ccache-current
+        ccache -o compiler_check=content
+        
+        if [ "${{ steps.safe-branch.outputs.name }}" == "${{ inputs.main_branch }}" ]; then
+          # default branch: use default branch cache
+          ccache --set-config=cache_dir="$HOME/.ccache-main"
```

Comment:
pedantry:

Maybe this should go to the existing
.github/actions/xahau-configure-ccache/action.yml

Reply:
https://github.com/tequdev/xahaud/pull/3